src/mame/drivers/rastersp.cpp

```diff
@@ -436,6 +629,193 @@ WRITE_LINE_MEMBER( rastersp_state::vblank_irq )
 		update_irq(IRQ_VBLANK, ASSERT_LINE);
 }
 
+/*
+	Mimic a Bacta Dataport unit just enough to satisfy Football Crazy.
+	Also mimic trackball unit. This uses a small PIC based interface board
+	which converts the trackball movement into RS232 data.
+	The data format is as follows:
+	Byte 0 : 0100abAB bits 7 & 6 = 01 signifies 1st byte
+	Byte 1 : 00CDEFGH bits 7 & 6 = 00 signifies 2 or 3rd byte (not identified)
+	Byte 2 : 00cdefgh
+
+	The trackball X movement is ABCDEFGH and Y movement abcdefgh
+	We do not have a PCB nor a dump of the PIC :(
+*/
```

Comment:
There’s already a stub BACTA data logger in src/mame/machine/bacta_datalogger.{h,cpp} – please consider using it rather than creating another stub device to mimic it.